notebooks/week1_test_package.py

```diff
@@ -0,0 +1,92 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Loan prediction with build package
+# MAGIC
+# MAGIC This notebook demonstrates how to predict loans  package build.
+
+# COMMAND ----------
+
+!pip install /Volumes/heiaepgah71pwedmld01001/lj_test/mlops_data/mlops_with_databricks-0.0.1-py3-none-any.whl
```

Comment:
Nice! so this will install your package in the notebook environment. You can also install this on your cluster by passing the same path to the Volume, this way, your package will always be there without needing to run !pip install.

Reply:
Yes, thanks Basak. I am sharing the cluster with others so I wanted to keep it isolated for now.